Common/Util/LeanData.cs

```diff
@@ -375,15 +393,15 @@ public static string GenerateZipEntryName(Symbol symbol, DateTime date, Resoluti
                 case SecurityType.Cfd:
                     if (isHourOrDaily)
                     {
-                        return string.Format("{0}.csv", 
+                        return string.Format("{0}.csv",
```

Comment:
Lots more white space changes. Please remove these -- they just make the diff and merge harder for others later on.